Sources/Utilities.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct Utilities {
+
+  /// Perform url encoding
+  ///
+  /// - Parameter string: The source string
```

Comment:
Also I'd name that parameter `allowedCharacters`, since just `allowed` is a bit ambiguous in this context.